examples/cluster/README.md

```diff
@@ -313,15 +317,16 @@ Changes to Outputs:
                   + region = "af-south-1"
                   + type   = "aws"
                 }
+              + connection_string = "couchbases://cb.irxmynm6vekhe5.cloud.couchbase.com" 
```

Comment:
Nit: We could also replace this string with something like the following:
couchbases://cb.xxxxxxxxxxxxxx.cloud.couchbase.com."

Reply:
I will make that change, note: I got this value from what we have in the example for [management api docs](https://docs.couchbase.com/cloud/management-api-reference/index.html#tag/Clusters/operation/getCluster)